src/components/posts/PostContent.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { dimensions } from 'styles/variables';
+
+interface ArticleHeaderProps {
+    dangerouslySetInnerHTML: {
+        __html: string;
+    } | undefined;
+}
```

Comment:
I suggestion a proposal. **optinal** is simple.
```suggestion
interface ArticleHeaderProps {
    dangerouslySetInnerHTML?: {
        __html: string;
    };
}
```

Reply:
Think it is better style
Thank you 👍

Reply:
Switch to resolved